draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1147,15 +1147,62 @@ Decoy digests increase the size of the SD-JWT. The number of decoy digests (or w
+credential presentations of the same user beyond the user's consent.
+Without unlinkability, an adversary might be able to learn more about the user than the user
+intended to disclose, for example:
+
+ * Cooperating Verifiers might want to track users across services to build
+   advertising profiles.
+ * Issuers might want to track where users present their credentials to enable
+   surveillance.
+ * After a data breach at multiple Verifiers, publicly available information
+   might allow linking identifiable information presented to Verifier A with
+   originally anonymous information presented to Verifier B, therefore revealing
+   the identities of users of Verifier B.
+
+The following types of unlinkability are considered here:
+
+ * Presentation Unlinkability: A Verifier should not be able to link two
+   presentations of the same credential.
+ * Verifier/Verifier Unlinkability: Two colluding Verifiers should not be able to
+   learn that they have received presentations of the same credential.
+ * Issuer/Verifier Unlinkability (Honest Verifier): An Issuer of a credential
+   should not be able to know that a user presented the credential to a certain
+   Verifier that is not behaving maliciously.
+ * Issuer/Verifier Unlinkability (Colluding/Compromised Verifier): An Issuer of a
+   credential should not be able to tell that a user presented the credential to
+   a certain Verifier, even if the Verifier colludes with the Issuer or becomes
+   compromised and leaks stored credentials from presentations.
+
+In all cases, unlinkability is limited to cases where the disclosed claims do
+not contain information that directly or indirectly identifies the user. For
+example, when a taxpayer identification number is contained in the disclosed claims, the Issuer and
+Verifier can easily link the user's transactions. However, when the user only
+discloses a birthdate to one Verifier and a postal code to another Verifier, the two Verifiers should not be able to determine that they were interacting with the same user.
+
+Issuer/Verifier unlinkability with a colluding or compromised Verifier cannot be
+achieved in salted-hash based selective disclosure approaches, such as SD-JWT, as the
+issued credential with the Issuer's signature is directly presented to the Verifier, who can forward it to
+the Issuer.
+
+Contrary to that, Issuer/Verifier unlinkability with an honest Verifier can generally be achieved.
+However, a callback from the Verifier to the Issuer, such as a revocation check, could potentially
+disclose information about the credential's usage to the Issuer.
+Where such callbacks are necessary, they MUST be executed in a manner that
+preserves privacy and does not disclose details about the credential to the Issuer. It is
+important to note that the timing of such requests could potentially serve as a side-channel.
+
+Verifier/Verifier unlinkablility can be achieved using batch issuance: A batch
```

Comment:
```suggestion
Verifier/Verifier unlinkablility and presentation unlinkablility can be achieved using batch issuance: A batch
```

Now that https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/354/commits/a1cbe276fcd1b5d85ac00a8985e3fd0fb7039f13 introduced presentation unlinkability and the flow of this whole section (where the unlinkablility types are first described and then discussed) it seems like it warrants a mention here. WDYT?